IMC/chatbot/Chatbot.md

```diff
@@ -0,0 +1,168 @@
+
+
+
+<img src="images/5.png" alt="img" style="zoom:50%;" />
+
+10. To use metrics for COVID-19, [enable the BigQuery API](https://console.cloud.google.com/flows/enableapi?apiid=bigquery) by selecting your project and agreeing to the terms of service.
+
+## Interacting with your agent
+
+### Using the Dialogflow Console Simulator
+
+<img src="images/6.png" alt="img" style="zoom:70%;" />
+
+
+
+1. Type your text query input in the [Dialogflow Simulator](https://cloud.google.com/dialogflow/docs/console#simulator).
+
+**Note: The custom payload of responses may not show up on the Dialogflow Console.**
+
+
+
+![img](images/7.png)
+
+
+
+### Using Dialogflow Messenger
+
+1. Go to the [Dialogflow Console](https://dialogflow.cloud.google.com/).
+2. Select your agent.
+3. Click **Integrations** in the left sidebar menu.
+4. Enable the [**Dialogflow Messenger**](https://cloud.google.com/dialogflow/docs/integrations/dialogflow-messenger) integration.
+5. Paste the embed code shown on the dialog in a webpage on your website. To interact with the agent on your site, click the icon at the bottom right, type your input query, and press Enter. The agent responds.
+6. [Customize the agent](https://cloud.google.com/dialogflow/docs/integrations/dialogflow-messenger#html_customizations) to appear the way you want it.
+
+### Using the Dialogflow API
+
+To interact with your agent using the Dialogflow API, ensure that your Google Cloud Platform project is properly [set up](https://cloud.google.com/dialogflow/docs/quick/setup), then submit a natural language query about COVID-19 and detect its intent.
+
+1. Create a request.json file that includes a natural language query about COVID-19. For example:
+
+   ```json
+    {
+      "query_input": {
+         "text": {
+             "text": "how to get tested",
+             "language_code": "en-US"
+           }
+        }
+      }
+   
+   // See the detectIntent page for details about the JSON format.
+   // page: https://cloud.google.com/dialogflow/docs/reference/rest/v2/projects.agent.sessions/detectIntent#request-body
+   ```
+
+<iframe src="https://medium.com/media/2732022c337c912f61e95a2781ebaf99" allowfullscreen="" frameborder="0" height="300" width="680" title="request.json" class="s t u dq ai" scrolling="auto" style="box-sizing: inherit; position: absolute; top: 0px; left: 0px; width: 680px; height: 299.984px;"></iframe>
+
+2. Run the following command to send a detect intent request to your agent.
+
+```json
+curl -X POST 
+
+        -H "Authorization: Bearer "$(gcloud auth application-default print-access-token) 
+
+        -H "Content-Type: application/json; charset=utf-8" 
+
+        -d @request.json 
+
+        https://dialogflow.googleapis.com/v2/projects/project-id/agent/sessions/session-id:detectIntent
+```
+
+**Note:** There are many [other ways](https://cloud.google.com/dialogflow/docs/quick/api#detect_intent) (C#, Go, Java, Python, Node.js, PHP, Ruby, etc) to interact with the API.
```

Comment:
If this is the end of the tutorial, include a conclusion to wrap things up! a few sentences is fine

Reply:
Added Conclusion